.github/workflows/autotools.yml

```
@@ -14,7 +29,24 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+    # test configure and build
     - name: configure
-      run: ./configure
+      run: ./configure --prefix=$PORTAUDIO_INSTALL_DIR
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:22: Double quote to prevent globbing and word splitting

(shellcheck)